projects/1.19/assets/create-planetary-tweaks/cp_tweaks/lang/zh_cn.json

```
@@ -0,0 +1,80 @@
+{
+   
+  "item.cp_tweaks.burning": "燃烧图标",
+  "item.cp_tweaks.warm": "温暖图标",
+  "item.cp_tweaks.comfy": "舒适图标",
+  "item.cp_tweaks.cold": "寒冷图标",
+  "item.cp_tweaks.freezing": "冰冻图标",
+  
+  "item.cp_tweaks.autumn": "秋季图标",
+  "item.cp_tweaks.dry": "干燥图标",
+  "item.cp_tweaks.spring": "春季图标",
+  "item.cp_tweaks.summer": "夏季图标",
+  "item.cp_tweaks.wet": "潮湿图标",
+  "item.cp_tweaks.winter": "冬季图标",
+  
+  "item.cp_tweaks.void_worm_gu": "虚空蠕虫黏液",
+  "item.cp_tweaks.steel_ingot": "钢锭",
+   
+  "item.cp_tweaks.unstable_ice_shard": "不稳定寒冰碎片",
+  
+  "item.cp_tweaks.void_wormed_sword": "虚空蠕虫剑",
+  "item.cp_tweaks.void_wormed_axe": "虚空蠕虫斧",
+  "item.cp_tweaks.void_wormed_shovel": "虚空蠕虫铲",

```

```suggestion
  "item.cp_tweaks.void_wormed_shovel": "虚空蠕虫锹",
```
Minecraft原版为`锹`
请注意全文修改

---

> Minecraft原版为`锹` 请注意全文修改

已全文修改